src/icons/scripts/convert.ts

```diff
@@ -155,6 +155,50 @@ function createCSSAttribute(css: string): types.JSXAttribute {
                     },
                   });
 
+                  // Replace `fill="none"` with `fill={fill}`
+                  traverse(jsx, {
+                    noScope: true,
+                    JSXAttribute({ node }) {
+                      if (
+                        node.name.type === "JSXIdentifier" &&
+                        node.name.name === "fill" &&
+                        node.value?.type === "StringLiteral" &&
+                        node.value.value === "none"
+                      ) {
+                        node.value = types.jsxExpressionContainer(
+                          types.identifier("fill")
+                        );
+                      }
+                    },
+                  });
+
+                  // Replace `stroke="currentColor"` with `fill={stoke}`
+                  traverse(jsx, {
+                    noScope: true,
+                    JSXAttribute({ node }) {
+                      if (
+                        node.name.type === "JSXIdentifier" &&
+                        node.name.name === "stroke" &&
+                        node.value?.type === "StringLiteral" &&
+                        (node.value.value === "currentColor" ||
+                          node.value.value === "#000" ||
+                          node.value.value === "#000000" ||
+                          node.value.value.toLowerCase() === "#12151a")
```

Comment:
minor: maybe some comments as to why we have these as special cases

Reply:
Yes, absolutely.